### ISSUE TITLE
How to make it less verbose during the build?

### ISSUE THREAD

**Issue by leoluz** [int128/gradle-swagger-generator-plugin#38]
Hi,

Nice plugin. Congrats!

Is is possible to make it less verbose? I see a bunch of the following logs during the gradle build:
```
[Daemon worker Thread 5] INFO io.swagger.codegen.AbstractGenerator - writing file ...
[Daemon worker Thread 5] WARN io.swagger.codegen.DefaultCodegen - Empty operationId found for path: ...

```
I tried to pass the `verbose=false` property in different places (plugin config, `additionalProperties` param and in the `configFile`) but none of them removes the logs above. 

Is it possible to disable these logs?

Thank you.

**Comment by int128** [comment]
Released 2.4.2. The generator task prints warnings and errors at default. Please try it.

**Comment by leoluz** [comment]
Its clean now. 
Thanks a lot!
